.zenodo.json

```diff
@@ -147,9 +153,17 @@
     {
       "name": "Hymers, Mark"
     },
+    {
+      "name": "Esteban, Oscar"
```

Comment:
@oesteban Please let me know if you would like your name presented differently, and if you have an affiliation/ORCID that you want associated with your contributions.

If you prefer not to be an author, let me know.

Reply:
```JSON
    {
      "affiliation": "Department of Psychology, Stanford University, CA, USA",
      "name": "Esteban, Oscar",
      "orcid": "0000-0001-8435-6191"
    }
```

Thanks very much!